linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/tests/authselect_modified_pam.fail.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# packages = authselect
+# platform = Red Hat Enterprise Linux 8,Red Hat Enterprise Linux 9,multi_platform_fedora
+# remediation = none
+
+SYSTEM_AUTH_FILE="/etc/pam.d/system-auth"
+
+# This modification will break the integrity checks done by authselect. The remediation
+# should be robust enough to deal with possible manual and unvalidated modification.
```

Comment:
This is comment is a bit outdated.
The remediation doesn't "deal with possible manual remediations", it aborts the remediation when it finds the authselect profiles have been edited by a third party.